cypress/plugins/index.js

```diff
@@ -0,0 +1,6 @@
+module.exports = (on, config) => {
+  require('cypress-mochawesome-reporter/plugin')(on);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/senacor/simple-consumer/pullRequest?prid=7519855)
```suggestion
  require("cypress-mochawesome-reporter/plugin")(on);
```